adapters/python/adapter_template.py

```
@@ -698,9 +698,224 @@ def _build_blutter_model(
+        return [found]
+    return None
+
+
+def _r2flutter_json(runner: List[str], target: str, flag: str):
+    """Run one r2flutter action and parse its JSON.
+
+    r2flutter emits one action per invocation and writes radare2 loader warnings to
+    stderr, so stdout is parsed on its own.
+    """
+    try:
+        proc = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)